### PR TITLE
Fix blank screen after in-app update on Windows

### LINKED ISSUE #460: Bug: After auto-updating APP screen stays blank
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.0.0-nightly.93

### What Operating System are you using?

Windows

### Operating System Version

11

### What arch are you using?

x64

### Last Known Working Ferdium version

6.0.0-nightly.91

### Expected Behavior

After an automatic in-app update the app starts itself and I can see my services.

### Actual Behavior

After an automatic in-app update the app starts itself but only a blank screen appears

### Steps to reproduce

1. Update through the in-app update feature
2. Let the update finish and the app start itself (automaticly) after
3. See the blank screen

### Debug link

_No response_

### Screenshots

image

### Additional information

This seems to be related with this PR https://github.com/ferdium/ferdium-app/pull/406 and this one  https://github.com/ferdium/ferdium-app/pull/444 (@cino), as I was not experiencing this before - altough I haven't tested it enough to comprove it.

Other changes on .92 (that seem to be unrelated to this):
* Upgrade 'macos-notification-state' to '2.0.1' by @vraravam in https://github.com/ferdium/ferdium-app/pull/440

Errors on console:
image
image
image

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Fix blank screen after in-app update on Windows

#### Motivation and Context
After discussing with @cino on how to resolve this issue, this PR fixes #460.
Turns out that after updating, the nsis installer on Windows opens the app with `--updated` as an argument, which is unwanted - given that the deeplink will try to route the app to `index.html#--updated`.

image

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes

Fix blank screen after in-app update on Windows

### CODE REVIEW

File: src/stores/AppStore.ts [hunk 1]
```diff
@@ -220,6 +220,7 @@ export default class AppStore extends TypedStore {
       if (!url) return;
 
       url = url.replace(/\/$/, '');
+      url = url.replace(/\s?--(updated)/, '');
```
Comment (vraravam) [inline src/stores/AppStore.ts:223]: Do we want to ensure this `updated` is at the end of the string - as opposed to being matched somewhere earlier/in the middle?

Comment (SpecialAro) [reply src/stores/AppStore.ts:223]: @cino what do you think about this?
I think that it should be match regardless of the position... but I don't know the direct implications of that.

Comment (cino) [reply src/stores/AppStore.ts:223]: Yeah that is fine, we can add that part as well if we are certain if it's always in the end and there Is nothing else behind it (a second option for example).

so in this case that would work, if electron changes that might not work anymore.

Comment (vraravam) [reply src/stores/AppStore.ts:223]: > so in this case that would work, if electron changes that might not work anymore.

exactly!

Comment (vraravam) [reply src/stores/AppStore.ts:223]: lets resolve this for now - and cross the bridge if we come to it later.

**Review by vraravam — APPROVED** [review]
LGTM

**Pull request merged**